.github/workflows/actions.yml

```diff
@@ -38,6 +38,8 @@ jobs:
         pip install torch>=2.0.1+cpu
         pip install "jax[cpu]"
         pip install keras-core
+        pip install keras-nlp-nightly --no-deps
+        pip install tensorflow-text==2.15
```

Comment:
If we use `keras-nlp-nightly`, should we also use `tensorflow-text-nightly`?  is there a reason to use `2.15` of text specifically?

Reply:
yes, tensorflow-text-nightly will pull in tf nightly and that version clashes with our keras 2 test env.